lms/djangoapps/teams/static/teams/js/spec/views/team_profile_spec.js

```diff
@@ -145,29 +167,76 @@ define([
+                                    'image_url_medium': '/image-url'
+                                }
+                            }
+                        },
+                            {
+                                'user': {
+                                    'username': 'bilbo1',
+                                    'profile_image': {
+                                        'has_image': true,
+                                        'image_url_medium': '/image-url'
+                                    }
+                                }
+                            },
+                            {
+                                'user': {
+                                    'username': 'bilbo2',
+                                    'profile_image': {
+                                        'has_image': true,
+                                        'image_url_medium': '/image-url'
+                                    }
+                                }
+                            }]
+                    });
+
+                    assertTeamDetails(view, 3, true);
+                    expect(view.$('.invite-header').text()).toContain('Invite Others');
+                    expect(view.$('.invite-text').text()).toContain('No invitations are available. This team is full.');
+                    expect(view.$('.invite-link-input').length).toBe(0);
+                });
+
+                it('can see & select invite url if team has capacity', function() {
+                    var requests = AjaxHelpers.requests(this);
+                    spyOn(TeamProfileView.prototype, 'selectText');
 
                     var view = createTeamProfileView(
-                        requests, { country: 'US', language: 'en', membership: DEFAULT_MEMBERSHIP}
+                        requests, {country: 'US', language: 'en', membership: DEFAULT_MEMBERSHIP}
                     );
                     assertTeamDetails(view, 1, true);
 
-                    expect(view.$(leaveTeamLinkSelector).length).toBe(1);
+                    expect(view.$('.invite-link-input').length).toBe(1);
 
-                    // click on Leave Team link under Team Details
-                    view.$(leaveTeamLinkSelector).click();
-
-                    // response to DELETE
-                    AjaxHelpers.respondWithNoContent(requests);
+                    view.$('.invite-link-input').click();
+                    expect(view.selectText).toHaveBeenCalled();
+                });
 
-                    // response to model fetch request
-                    AjaxHelpers.respondWithJson(requests, createTeamModelData({country: 'US', language: 'en'}));
+                it('can leave team successfully', function() {
+                    var requests = AjaxHelpers.requests(this);
 
+                    var view = createTeamProfileView(
+                        requests, { country: 'US', language: 'en', membership: DEFAULT_MEMBERSHIP}
```

Comment:
This is a stupid nit that shouldn't block this from merging, but above there's no space between `{` and `country`.